examples/haas.cfg

```diff
@@ -62,10 +62,20 @@ endpoint = http://127.0.0.1:5000
 
 [database]
 # A SQLAlchemy database URI, specifying the database for the server to
-# connect to. At present, only SQLite and PostgreSQL are supported.
+# connect to. At present, only SQLite and PostgreSQL are supported, and only
+# PostgreSQL is supported by our migration scripts.
+#
+# If a relative path is used with SQLite, it will be relative to the HaaS source
+# tree. Unless you're doing development, this is almost certainly not what you
+# want, so use an absolute path.
+#
+# Use the sqite database file ``/absolute/path/to/haas.db``:
+uri = sqlite:////absolute/path/to/haas.db
```

Comment:
@henn, there's already a postgres example commented out a few lines below.

Again, no strong feelings on this part; come to a consensus and I'll make the change.